fix: Updating txt files for class09_pb3_hystRelperm

arng40 opened the pull request:
In both deck  `class09_pb3_hystRelperm_direct_base.xml`  & `class09_pb3_hystRelperm_iterative_base.xml`, 
wrong files have been set in `TableFunction` for `imbibitionWaterRelativePermeabilityTable`.
Update drainage files for imbibition files.

arng40 committed:
set the correct files

arng40 committed:
fix for direct_base

arng40 committed:
Merge branch 'develop' into bugfix/dudes/fix-hysterisis-deck

arng40 committed:
xsd

arng40 committed:
Merge branch 'bugfix/dudes/fix-hysterisis-deck' of https://github.com/GEOS-DEV/GEOS into bugfix/dudes/fix-hysterisis-deck

paveltomin committed:
Merge branch 'develop' into bugfix/dudes/fix-hysterisis-deck

arng40 commented:
The modfied deck isn't converging and the [example ](https://geosx-geosx.readthedocs-hosted.com/en/latest/docs/sphinx/advancedExamples/validationStudies/carbonStorage/isothermalHystInjection/Example.html?_sm_au_=iVVjqpW6F6JM74WFQ0WpHK6H8sjL6) related to this deck need to be changed.

arng40 committed:
xsd

arng40 committed:
Merge branch 'bugfix/dudes/fix-hysterisis-deck' of https://github.com/GEOS-DEV/GEOS into bugfix/dudes/fix-hysterisis-deck

Commit:
tune the case

Commit:
Merge branch 'bugfix/dudes/fix-hysterisis-deck' of https://github.com/GEOS-DEV/GEOS into bugfix/dudes/fix-hysterisis-deck

arng40 committed:
space

arng40 committed:
Merge remote-tracking branch 'origin/develop' into bugfix/dudes/fix-hysterisis-deck

paveltomin commented:
@arng40 were you able to test and verify after my changes? is it good to move into the merge queue?

arng40 commented:
Yep I tested in paraview and the GIF looks like the one in the doc

arng40 committed:
Merge remote-tracking branch 'origin/develop' into bugfix/dudes/fix-hysterisis-deck

paveltomin committed:
Merge branch 'develop' into bugfix/dudes/fix-hysterisis-deck

paveltomin committed:
Merge branch 'develop' into bugfix/dudes/fix-hysterisis-deck

paveltomin committed:
Update baseline path in integrated tests configuration

paveltomin committed:
Update BASELINE_NOTES.md with new PR entries

Added entries for pull requests #3788 and #3621 with corresponding dates and justifications.
